src/mcp/shared/httpx_utils.py

```diff
@@ -0,0 +1,64 @@
+"""Utilities for creating standardized httpx AsyncClient instances."""
+
+from __future__ import annotations
+
+from typing import Any
+
+import httpx
+
+__all__ = ["create_mcp_http_client"]
+
+
+def create_mcp_http_client(
+    headers: dict[str, Any] | None = None,
```

Comment:
```suggestion
    headers: dict[str, str] | None = None,
```

ref.: https://github.com/encode/httpx/blob/6c7af967734bafd011164f2a1653abc87905a62b/httpx/_models.py#L139